architecture/environments/operator.md

```diff
@@ -13,15 +13,15 @@ compiled to [Go
 structs](https://github.com/istio/api/blob/master/operator/v1alpha1/operator.pb.go).
 `IstioOperatorSpec` has pass-through fields to the Helm values.yaml API, but these are additionally validated through
 a [schema](../operator/pkg/apis/istio/v1alpha1/values_types.proto).
-1. [Controller](#k8s-controller) code. The code comprises the K8s listener, webhook and logic for reconciling the cluster
+2. [Controller](#k8s-controller) code. The code comprises the K8s listener, webhook and logic for reconciling the cluster
```

Comment:
There are a lot of README changes, and these sequence numbers should not be changed, nor should the table format. They will automatically be formatted.